src/controllers/playback/video/index.html

```diff
@@ -87,6 +87,9 @@ <h3 class="osdTitle"></h3>
                 <button is="paper-icon-button-light" class="btnVideoOsdSettings autoSize" title="${Settings}">
                     <span class="largePaperIconButton material-icons settings" aria-hidden="true"></span>
                 </button>
+                <button is="paper-icon-button-light" class="btnPlayerStats autoSize" title="${PlaybackData}">
```

Comment:
I've placed the new info button/icon to the right of the settings button. That used to be the right-most button on TV, in the bottom-right corner; that's the info now. The number of key presses to get to the settings button is hence unchanged, so muscle memory shouldn't be affected.